### ISSUE TITLE
Enforce consistency of LES_driven_SCM

### ISSUE THREAD

**Issue by ilopezgp** [CliMA/TurbulenceConvection.jl#505]
The current definition of the LES_driven_SCM,

1. **Initializes** the SCM profiles as the average over the last 6 hours in LES. [Cases.jl]
2. Applies the same horizontal and vertical eddy **advection tendency** as LES, since these are constant in time in LES as well. [Cases.jl]
3. Applies a **relaxation** to the average LES momentum profiles from the last 6 hours with a 6 hour timescale, consistent with the LES simulations (which relax to the GCM profiles). This means that this is the only time-dependent tendency that we have. [Cases.jl]
4. Instead of using a **relaxation** term for temperature and q_t, we instead apply the last 6-hour average relaxation tendency from LES. This is no longer a relaxation term, because it induces a tendency based on the mean difference between the GCM and the LES, none of which are the SCM.
5. Applies the average **radiative** heating rates over the last 6 hours of LES to the SCM.
6. Applies the average **surface** conditions over the last 6 hours of LES to the SCM.
7. The simulations are then run for 12 hours. This is fine only for statistical steady-state solutions. If the boundary layer is growing, for example, then we cannot really compare the simulations. 

I think most of these points are OK. However, I think 3. and 4. are inconsistent, and I think we should correct 4. to be similar to 3., using the relaxation timescale in equation (9) of Shen et al. For calibration, point 7 is **very** important. We can only do this for simulation that are in statistical steady-state.

**Comment by ilopezgp** [comment]
I think the forcing is steady-state, but that need not imply that the state is. For example, if we have any simulation without clouds, surface heating cannot be compensated by cloud-top radiative cooling and the boundary layer would grow (similar to Soares, Nieuwstadt). If you compare different times between LES and SCM, you will get different results and bias your calibration process!

**Comment by ilopezgp** [comment]
I agree with @szy21 in that assuming steady-state can be detrimental in the future. A small fix that could go a long way is the following,

- Initialize the SCM profiles with the LES profiles at `t_end - 6h` (maybe we can average from `t_end - 6.15 h` to `t_end - 5.45 h` to smooth the profiles).
- Run the simulations only for 6 hours in order to calibrate using the last ~3 hours. Three hours of spin up should be more than enough for an SCM in my opinion, @costachris can confirm for these particular cases.

This setup is not perfect, but since we start from the same place and we are inputting the same energy (we input the average surface and radiative fluxes from LES), we should have very close statistics over 3 hours from LES and SCM. And of course, fix 4.

**Comment by costachris** [comment]
So it sounds like the immediate priorities are to a) Use an average 30 mins around `t_end` - 6 to initialize the SCM + run for 6 hours and b) fix 4. 

And I agree 3 hours should be enough for spin-up based on what I've seen .

**Comment by ilopezgp** [comment]
After our meeting with Tapio: The plan forward is approved, pending revisiting the surface fluxes in the future (which Tapio would prefer to be fixed SST with interactive fluxes), and using RRTMG interactively.

**Comment by yairchn** [comment]
> After our meeting with Tapio: The plan forward is approved, pending revisiting the surface fluxes in the future (which Tapio would prefer to be fixed SST with interactive fluxes), and using RRTMG interactively.

we could use the current coded 'SurfaceMoninObukhov' function (currently unused as GABLS is using its dry version) and have it ready before we use SurfaceFluxes.jl. I am happy to do that we myself

**Comment by costachris** [comment]
The SCM now uses a relaxation to LES profiles for `theta` and `qt` in the same manner as Zhaoyi's paper. Initial conditions for the SCM are the 1-hour LES mean around `t_end` - 6 hours, and the SCM simulation is run of 6 hours. Addressing points 4, 7. #511

**Comment by costachris** [comment]
#1298 nudges thetali, qt, U, and V to the GCM profile (initial LES profile)
